yaml/stopwords_id.yml

```diff
@@ -0,0 +1,66 @@
+        - [saya, aku, gue, sendiri, kita, kami, anda, kamu, engkau, saudara, milikmu, dirimu, dia, ia, beliau, kepunyaannya, punya*]
+        # [it, itself, they, them, theirs, themselves, this, that, these, those]
+        - [itu, ini, mereka, bahwa*, yang]
+    possesive: 
+        # [my, our, your, his, her, its, their][my, our, your, his, her, its, their]
+        - [milikku, milik saya, milik*]
+    interogative: 
+        # [what, which, who, whom, whose, when, where, why, how]
+        - [apa, mana, siapa*, kapan, ketika, mengapa, kenapa, bagaimana]
+    contraction: 
+        # [i'm, you're, he's, she's, it's, we're, they're, i've, you've, we've, they've, i'd, you'd, he'd, she'd, we'd, they'd, i'll, you'll, he'll, she'll, we'll, they'll]
+    
+verb: 
+    basic: 
+        # [am, is, are, was, were, be, been, being, have, has, had, having, do, does, did, doing]
+        - [adalah, dulu, menjadi,berada, telah, makhluk, melakukan, berbuat, dilakukan, perbuatan]
+    modal: 
+        # [would, should, could, ought, will]
+        - [akan, harus, bisa, seharusnya]
+    contraction: 
+        # [isn't, aren't, wasn't, weren't, hasn't, haven't, hadn't, doesn't, don't, didn't, won't, wouldn't, shan't, shouldn't, can't, cannot, couldn't, mustn't, let's, that's, who's, what's, here's, there's, when's, where's, why's, how's]
+    
+    reporting: 
+        # [say, says, said, tell, tells, told]
+        - [mengatakan, kata*, berkata, mengucapkan, berucap, ujar, berpendapat, bilang, membilangkan, mengomong, memprikan,mengira, menceritakan, menyuruh, bercerita, mengetahui, mengenal*, menunjukkan]
+        # [report, reports, reported]
+        - [jelaskan, deberitahu, melapor*, mengabarkan, memberitahu*, mewartakan, lapor*, merencana, repot, dilaporkan]
+article: 
+    # [a, an, the]
+    - [sebuah, sebatang, sang]
+conjunction:
+    # [and, but, if, or, because, so, while, nor]
+    - [dan, serta, dengan, tambah, tetapi, melainkan, jika, atau, karena, sebab, begitu, amat, jadi, demikian, sementara, sedangkan, sambil, meski*, walaupun, maupun, namun]
+adverb: 
+    # [as, until, once, here, there, all, any, both, each, few, many, more, most]
+    - [sebagai, seperti, ajak, sampai, sehingga, hingga, sekali, sini, sana, semua, segenap, segela, sekalian, sesuatu, kedua*, tiap*, masing*, banyak, lebih banyak]
+    # [other, some, such, "no", not, only, then, too, very, little, less]
+    - [lain, beberapa, hanya, "tidak", "bukan", saja, cuma, sangat, sedikit, tidak banyak]
+preposition: 
+    # [of, at, by, for, with, about, against, between, into, through, during]
+    - [di, oleh, dengan, menurut, untuk, bagi, dengan, pada, tentang, terhadap, antara, di antara, ke dalam, melalui, melewat*]
+    # [before, after, above, below, to, from, up, down, in, out, "on", "off", over, under, again, further, than]
+    - [sebelum, stelah, atas, bawah, ke, dari, naik, turun, dalam, luar, di atas, di bawah,  lagi, lebih lanjut, daripada]
```

Comment:
Should be "lebih, lanjut"?

Reply:
"lebih lanjut" is a phrase and it is the most suitable translation of "further", but I understand that multi-words expression is uncommon in stopwords list,  so I can replace with "lagi".

Reply:
You can include "lebih" and "lanjut" if they do not destroy other phrases.